### DISCUSSION TITLE
Unable to access my student developer pack

### DISCUSSION THREAD

**Discussion by kky5082** [community/community#66593] [GitHub Education]
### Hi everyone,

Hi, I am a registered student user. My student developer pack was approved yesterday, but I still didn't have access to it.
This is the screenshot of my application.
https://x.imgs.ovh/x/2023/09/10/64fca20f764c0.png

**Comment by arafajs** [comment]
It usually takes 72 hours for your GitHub Student Developer Pack benefits to take effect after activation. However, during busy periods, such as the start of a new semester, it may take a bit longer.

**Comment by mantasu** [comment]
Hey, I had the same issue, you may still have to wait for some time since yeah there will be some delay before your benefits become active. When "Approved" tag becomes purple, you should be ready:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

Apologies if you already received it but if not and it has already been 2-3 weeks, you may want to open a [ticket](https://support.github.com/request/education?tags=docs-generic)
